vlog4j-client/src/test/java/org/vlog4j/client/picocli/PrintQueryResultsTest.java

```diff
@@ -20,14 +20,20 @@
  * #L%
  */
 
+import static org.junit.Assert.assertEquals;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+
 import org.junit.Test;
 import org.vlog4j.client.picocli.PrintQueryResults;
 import javax.naming.ConfigurationException;
 
 public class PrintQueryResultsTest {
+	String outputConfigurationBase = "  --print-query-result-size: %b\n  --print-complete-query-result: %b\n";
 
 	@Test
-	public void validate_completeFalseSizeTrue_valid() throws ConfigurationException {
+	public void validate_sizeTrueCompleteFalse_valid() throws ConfigurationException {
 		// default configuration
 		PrintQueryResults prq = new PrintQueryResults();
 		prq.setSizeOnly(true);
```

Comment:
there is nothing tested here (just that an exception is not thrown, but I don't think this is the way to do it). You can extract the boolean expression that decides whether a config is valid into a package-protected isValid() method, and test assert that isValid() returns true for valid configurations, and that isValid() returns false and validate() throws an exception for non valid configurations.